sci-biology/hmmer/hmmer-3.1_beta2.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils
 MY_PV="3.1b2"
```

Comment:
I know that this was already in the ebuild, but if you send a pull request its welcome if you could further improve (this part of) the ebuild :)
The naming of the ebuild is fine, see https://devmanual.gentoo.org/ebuild-writing/file-format/. However, `MY_PV="3.1b2"` could rewritten using the versionator eclass or using a bash substitution. Please take a look at *Version and Name Formatting Issues* in https://devmanual.gentoo.org/ebuild-writing/variables/. The versionator should be smart enough to handle the translation, but I am not using it that regularly. Please give it a try.